app/models/motorcycle.rb

```diff
@@ -0,0 +1,4 @@
+class Motorcycle < ApplicationRecord
+  has_one_attached :image, dependent: :destroy
+  belongs_to :user
+end
```

Comment:
- Please, add required validations for elements on this model. 🐶

Reply:
Hello @vichuge, Thank you for the suggestions. We will try to implement as many of them as we can. 
Regarding the tests, we used the command `rake swag:specs:swaggerize` and all tests passed. The tests were to help us generate the API documentation. Thank you